pkg/agent/nodeportlocal/portcache/port_table.go

```diff
@@ -47,14 +42,13 @@ type NodePortData struct {
 	PodPort  int
 	PodIP    string
 	Protocol ProtocolSocketData
+	// defunct is used to indicate that a rule has been partially deleted: it is no longer
+	// usable and deletion needs to be re-attempted.
+	defunct bool
```

Comment:
It looks `defunct` is an in-memory flag, so it would be lost after a restart on the agent? Is it as expected?

Reply:
yes - after we restart we sync all the NPL rules and there is no issue

Reply:
thanks for the clarification.